ui/apos/components/AposExportModal.vue

```diff
@@ -175,8 +179,11 @@ export default {
 
   computed: {
     moduleLabel() {
-      const moduleOptions = window.apos.modules[this.moduleName];
-      const label = this.checked.length > 1 ? moduleOptions.pluralLabel : moduleOptions.label;
+      const moduleOptions = apos.modules[this.moduleName];
+      const label = this.checked?.length > 1
```

Comment:
```suggestion
      const label = this.count > 1
```

if possible to use the computed below 👇